app/lib/actions/foreman/host/import_facts.rb

```diff
@@ -8,7 +8,13 @@ def resource_locks
 
         def plan(_host_type, host_name, facts, certname, proxy_id)
           facts['domain'].try(:downcase!)
-          host = ::Host::Base.import_host(host_name, certname, facts, proxy_id)
+          host = if SETTINGS[:version] > '1.16'
+                   ::Host::Base.import_host(host_name, certname, proxy_id)
+                 else
+                   # backwards compatibility
+                   ::Host::Managed.import_host(host_name, certname, facts, proxy_id)
```

Comment:
I've merged prematurely: The previous signature in the foreman was: `import_host(hostname, import_type, certname = nil, proxy_id = nil)`, please update correspondingly